importers/audio_object.py

```diff
@@ -312,7 +312,7 @@ def get_mw_audio_1track(key: str) -> [CE_AudioObject]:
                         person.description = ppl['description']
 
                     elif prov_name == 'WIKIPEDIA_NL':
-                        ext_link = 'https://nl.wikipedia.org/wiki/{}'.format(ext_link)
+                        wiki_data_link = 'https://en.wikipedia.org/wiki/{}'.format(ext_link)
```

Comment:
I'm not sure about this change - it looks like 'WIKIPEDIA_NL' is a specific link to nl.wikipedia.org?
Double-check here as well when creating the Person, it'd be good to set the language field to dutch (as well as setting it to english for the english wikipedia links)

Reply:
my bad